### DISCUSSION TITLE
"Heresies" as accepted parts of bS

### DISCUSSION THREAD

**Discussion by kreincke** [bootscore/bootscore#353] [General]
Recently the bS-Team successfully helped me to activate a multi-level menu. In that respective pull-request process I've learned that bS wants to be / become a theme for the pure bootstrap library (hopefully I've understood that correctly) and that therefore multi level menus cannot literally be integrated into bS. The only way is to have an external howto by which one collect the necessary elements from different places and add them manually to ones on bS-child theme.

I totally agree with the aim to stay as close as possible at bootstrap - as base line. But couldn't it be acceptable to have a directory like 'heresies' inside of bS, which contains all elements and howtos to setup such deviant features like for example multi-level menus? The advantage would be to have all information & elements at one place without disturbing the base line itself?

If you agree with this I would a such a directory and initialize it with the multi-level menu as it was recently discussed here.

best regards KR

**Comment by justinkruit** [comment]
I'm happy with the effect you're putting into bootScore, thanks for that! I currently can't respond as long as you, so I just want to apologize for that as I feel like I should spend more time on the response.

I personally think it's better to have the theme as clean as possible. Having a multi level menu is an addition to Bootstrap that isn't native and doesn't feel right to be added to the theme. I like the idea of having additional information about how to enable such stuff, but I think the best usage of this would be in a wiki or something instead of having them in the theme by default. They should be an additional choice that isn't downloaded on every bootScore version.

Thank you again for your input! 😄

**Reply by kreincke** [reply]
Ok. Got it. Also this is a reasonable position. Hence, I will find an external way to collect 'aspects' by which bS could be extended if the user wants to incorporate one of them. To reach that aim, I will at fist complete my child theme with all the things I need. And then I will release those aspects which perhaps could be interesting for other people. Again: thanks to all of you, you doing a great job.
